docs/source/user_guide/configuration.md

```diff
@@ -0,0 +1,47 @@
+# Configuration
+
+For a complete list of configuration options, see :link: ./env_vars :link-type: doc
```

Comment:
This link didn't render properly, and I think it should be done using one of the options [here](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html).

Reply:
Thanks! I didn't know how links work here and just tried to follow an example from vllm